vresutils/load.py

```diff
@@ -137,35 +137,47 @@ def timeseries_opsd(years=slice("2011", "2015"), fn=None):
         load = load.loc[years]
 
     # manual alterations:
-    # GB input given in 3 regions, there is no "global one":
+    load = manual_alternations_opsd(load)
+
+    return load
+
+def copy_timeslice(load, cntry, start, stop, delta):
+    start = pd.Timestamp(start)
+    stop = pd.Timestamp(stop)
+    if start in load.index and stop in load.index:
+        load.loc[start:stop, cntry] = load.loc[start+delta:stop+delta, cntry].values
+    return load
+    
+
+def manual_alternations_opsd(load):
+    # GB in the input is split in 3 regions:
     # GBN (Great Britain), NIR (northern ireland), together forming
-    # UKM (united kingdom). Therefore, we choose UKM to be global.
+    # UKM (united kingdom). Therefore, we choose only UKM.
     # the sum of GBN and NIR seems incomplete, more data is missing
     # interpolate the rest (copying from previous weeks might be better)
     load['GB'] = load['GB_UKM']
-    load['GB'] = load['GB'].interpolate()
     load = load.drop(columns=['GB_GBN', 'GB_NIR', 'GB_UKM'])
     
+    # To fill loder periods of gaps (more than 4 hours), we copy a period before into it
```

Comment:
When Aarhus did some easy load fixing earlier, they found it is important to NOT mix weekdays and weekends. And ideally also account for holidays.

Did you check this for the copied slices?

Reply:
I think accounting for holidays is hard(er) as they differ in each country and each reagion, but I fixed one weekend. Thanks for the comment!

Reply:
Fine enough